### PR TITLE
Publish VSIX installer with build

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

**Comment by sharwell** [comment]
@jaredpar for approval

File: azure-pipelines.yml [hunk 1]
```diff
@@ -152,6 +152,14 @@ jobs:
         publishLocation: Container
       condition: succeeded()
 
+    - task: PublishBuildArtifacts@1
+      displayName: Publish VSIX Packages
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\VSSetup\Release\Installer'
+        ArtifactName: 'VSIX - PreRelease'
```
Comment (tmat) [inline azure-pipelines.yml]: `Installer`?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -152,6 +152,14 @@ jobs:
         publishLocation: Container
       condition: succeeded()
 
+    - task: PublishBuildArtifacts@1
+      displayName: Publish VSIX Packages
```
Comment (tmat) [inline azure-pipelines.yml]: Shouldn't this rather say `Publish installer`?

Comment (sharwell) [reply azure-pipelines.yml]: I don't mind the current name, if for no other reason than the install script wrapping the VSIX package is only needed as a workaround for limitations (arguably bugs) in the VSIX installer/uninstaller. I also wouldn't object if someone sent a PR to rename this.

**Pull request merged**